algebird-core/src/main/scala/com/twitter/algebird/SummingCache.scala

```diff
@@ -58,7 +58,7 @@ class SummingCache[K, V](capacity: Int)(implicit sgv: Semigroup[V])
   }
   override def flush: Option[Map[K, V]] = {
     // Get a copy of the cache, since it is mutable
-    val res = optNonEmpty(Map(cache.toSeq: _*))
+    val res = optNonEmpty(cache.iterator.toMap)
```

Comment:
was there a particular reason to make this change? It looks like its probably more efficient, but curious if you had a benchmark/profile you saw this in?